compiler/tests-spec/testData/diagnostics/linked/expressions/try-expression/p-1/neg/4.1.fir.kt

```diff
@@ -14,7 +14,7 @@ fun case1() {
         throwException()
     } catch (e: ExcA) {
     } finally {
-    } <!UNRESOLVED_REFERENCE!>catch<!> (<!UNRESOLVED_REFERENCE!>e<!><!SYNTAX!><!SYNTAX!><!>: ExcB)<!> {
```

Comment:
Wow... unresolved `catch`... what on earth has happened here?! Was it really `catch` clause that was not resolved (no way!) or was it a bug on `source` selection?

Reply:
I think it's the syntax issue since the `catch` is placed after `final`. I think this is WAI.